# HG changeset patch
# User andrew
# Date 1365768981 -3600
# Node ID 8f5ad2cacf6fff8efd817c5373469037d945f3dc
# Parent  3c7eb39c811464c98847a4f1a167636c1db80f17
8003445: Adjust JAX-WS to focus on API
Reviewed-by: vinnie, ahgross, mgrebac

diff --git a/src/share/lib/security/java.security b/src/share/lib/security/java.security
--- jdk/src/share/lib/security/java.security
+++ jdk/src/share/lib/security/java.security
@@ -150,7 +150,8 @@
                com.sun.org.apache.xalan.internal.xsltc.util.,\
                com.sun.org.apache.xml.internal.res.,\
                com.sun.org.apache.xml.internal.serializer.utils.,\
-               com.sun.org.apache.xml.internal.utils.
+               com.sun.org.apache.xml.internal.utils.,\
+               com.sun.org.glassfish.
 
 #
 # List of comma-separated packages that start with or equal this string
@@ -185,7 +186,8 @@
                    com.sun.org.apache.xalan.internal.xsltc.util.,\
                    com.sun.org.apache.xml.internal.res.,\
                    com.sun.org.apache.xml.internal.serializer.utils.,\
-                   com.sun.org.apache.xml.internal.utils.
+                   com.sun.org.apache.xml.internal.utils.,\
+                   com.sun.org.glassfish.
 
 #
 # Determines whether this properties file can be appended to
diff --git a/src/share/lib/security/java.security-solaris b/src/share/lib/security/java.security-solaris
--- jdk/src/share/lib/security/java.security-solaris
+++ jdk/src/share/lib/security/java.security-solaris
@@ -151,7 +151,8 @@
                com.sun.org.apache.xalan.internal.xsltc.util.,\
                com.sun.org.apache.xml.internal.res.,\
                com.sun.org.apache.xml.internal.serializer.utils.,\
-               com.sun.org.apache.xml.internal.utils.
+               com.sun.org.apache.xml.internal.utils.,\
+               com.sun.org.glassfish.
 
 #
 # List of comma-separated packages that start with or equal this string
@@ -186,7 +187,8 @@
                    com.sun.org.apache.xalan.internal.xsltc.util.,\
                    com.sun.org.apache.xml.internal.res.,\
                    com.sun.org.apache.xml.internal.serializer.utils.,\
-                   com.sun.org.apache.xml.internal.utils.
+                   com.sun.org.apache.xml.internal.utils.,\
+                   com.sun.org.glassfish.
 
 #
 # Determines whether this properties file can be appended to
diff --git a/src/share/lib/security/java.security-windows b/src/share/lib/security/java.security-windows
--- jdk/src/share/lib/security/java.security-windows
+++ jdk/src/share/lib/security/java.security-windows
@@ -151,7 +151,8 @@
                com.sun.org.apache.xalan.internal.xsltc.util.,\
                com.sun.org.apache.xml.internal.res.,\
                com.sun.org.apache.xml.internal.serializer.utils.,\
-               com.sun.org.apache.xml.internal.utils.
+               com.sun.org.apache.xml.internal.utils.,\
+               com.sun.org.glassfish.
 
 #
 # List of comma-separated packages that start with or equal this string
@@ -186,7 +187,8 @@
                    com.sun.org.apache.xalan.internal.xsltc.util.,\
                    com.sun.org.apache.xml.internal.res.,\
                    com.sun.org.apache.xml.internal.serializer.utils.,\
-                   com.sun.org.apache.xml.internal.utils.
+                   com.sun.org.apache.xml.internal.utils.,\
+                   com.sun.org.glassfish.
 
 #
 # Determines whether this properties file can be appended to
